tools/assignmentPlacer.py

```diff
@@ -0,0 +1,28 @@
+import os, sys
+import shutil
+
+gradeable_path = '/var/local/submitty/courses/s20/sample/submissions/grades_released_homework_autohiddenEC'
```

Comment:
It would be good to have this be customizable via args. I think for a v1, we can probably just expect the user to provide the full path they want to copy to.